src/main/java/com/redhat/devtools/lsp4ij/launching/templates/LanguageServerTemplateManager.java

```diff
@@ -131,6 +135,7 @@ public LanguageServerTemplate createLsTemplate(@NotNull VirtualFile templateFold
         String templateJson = null;
         String settingsJson = null;
         String initializationOptionsJson = null;
+        String clientSettingsJson = null;

```

Comment:
The real changes in this file begin here where we add handling for `clientSettings.json`.